app/component/navigation/toggle-map-tracking.cjsx

```diff
@@ -0,0 +1,25 @@
+React           = require 'react'
+Icon            = require '../icon/icon'
+cx              = require 'classnames'
+
+
+class ToggleMapTracking extends React.Component
+  @propTypes:
+    enableMapTracking: React.PropTypes.func.isRequired
+    disableMapTracking: React.PropTypes.func.isRequired
+    tracking: React.PropTypes.bool.isRequired
+
+  handleClick: =>
+    if !@props.tracking
+      @props.enableMapTracking()
+    else
+      @props.disableMapTracking()
+
+  render: ->
+    <div className="toggle-positioning-container" onClick={@handleClick}>
+      <Icon img={'icon-icon_mapMarker-location'}
+            className={cx "icon-mapMarker-toggle-positioning-online": @props.tracking,
```

Comment:
This starts to be nit-picking, but this could be made more understandable(?) by replacing with `className={"icon-mapMarker-toggle-positioning-" + if @props.tracking then "online" else "offline"}`